packages/core/src/compile.ts

```diff
@@ -247,25 +246,32 @@ export async function generateRuntime(loadedTranslation: LoadedTranslation) {
   await writeIfChanged(outputFilePath, declaration);
 }
 
-export function watch(config: UserConfig) {
+export async function watch(config: UserConfig) {
```

Comment:
This function already returned a promise, but it now needs to be explicitly async so I can do the dynamic import below.